pkg/vault/client.go

```diff
@@ -36,6 +36,7 @@ func NewClient(config *Config, token string) (Client, error) {
 	if config == nil {
 		config = &Config{Config: api.DefaultConfig()}
 	}
+	log.Entry().Debugf("Login to vault %s in namespace %s successfull", config.Address, config.Namespace)
```

Comment:
Is this really the point where a successful vault login is confirmed?
From reading the code, it looks like it can't be earlier than where a token is set (e.g. line 49).